opensearchpy/_async/http_aiohttp.py

```diff
@@ -97,7 +98,7 @@ def __init__(
         http_compress=None,
         opaque_id=None,
         loop=None,
-        **kwargs,
+        **kwargs
```

Comment:
nit: Was this change introduced by styling tool? If not, can you revert here and other places?  Thanks